### ISSUE TITLE
Quick Screenshot Plugin not working

### ISSUE THREAD

**Issue by BTW-l-Use-Arch** [xournalpp/xournalpp#4886]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: gruda linux
 - (Linux only) Desktop environment: Gnome-Wayland
 - Which version of libgtk do you use - 3.24.37
     - can be obtained by running `xournalpp --version` or in the **About** window
 - Version of Xournal++: 1.1.3
     - can be obtained by running `xournalpp --version` or in the **About** window
 - Installation method: [e.g. PPA, Flatpak, official release, etc.]

**Describe the bug**
the Quick screenshot plugin not working

**To Reproduce**
1. Step 1 - enable the plugin
2. Step 2 - try to use (ie: through the menu or through the shortcut (ctrl+alt+T)) it nothing will happen
3. etc.

**Expected behavior**
First time using this Plugin so not much idea but at least a screenshot window or past option should open but nothing at all happens

**Screenshots of Problem**
(Optional) If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other information or context about the problem here.

**Comment by rolandlo** [comment]
Do you have `maim` and `xclip` installed (or `scrot` as an alternative)? Do you get any console messages if starting Xournal++ from the terminal?

**Comment by BTW-l-Use-Arch** [comment]
I don't know what they are, if they are additional plugins than no.
I never tried opening it though the terminal until now but...
![image](https://github.com/xournalpp/xournalpp/assets/84948029/f4c20c2b-71e3-4367-bac3-24cb0322961c)
![image](https://github.com/xournalpp/xournalpp/assets/84948029/3d366b0f-4ee5-45da-96ea-201c59e2ef0d)

**Comment by rolandlo** [comment]
1) It's `xournalpp`
2) `maim` and `scrot` take screenshots, `xclip` is a tool for dealing with the clipboard. The plugin doesn't make screenshots on its own, it just executes a tool you have installed.

**Comment by BTW-l-Use-Arch** [comment]
![image](https://github.com/xournalpp/xournalpp/assets/84948029/1bfc1493-1b7c-4ea2-907c-b896a6055b19)
didn't work i restarted the app btw I downloaded this through discover/flathub if that matters

**Comment by rolandlo** [comment]
That does matter indeed. First you start Xournal++ via `flatpak run com.github.xournalpp.xournalpp` and secondly there are sandboxing issues preventing screenshot tools like `maim` or `scrot` to be used.

**Comment by BTW-l-Use-Arch** [comment]
Bro just love the way you 
anyway you were right
![image](https://github.com/xournalpp/xournalpp/assets/84948029/373323f4-f077-4cae-b83d-2f618a1c0e09)
the problem now is that I don't know how am I supposed to install scrot
btw sorry for the late reply had exams

**Comment by rolandlo** [comment]
I don't think it will work with the flatpak. It can't access the screenshot tool even if you install it on your system.

**Comment by BTW-l-Use-Arch** [comment]
thanks I'll just install the app another way like

**Comment by defarm** [comment]
Similar issue, perhaps?

    OS: Ubuntu 22.04 (Focal)
    (Linux only) Desktop environment: Gnome-Wayland
    Which version of libgtk do you use - 3.24.38
        can be obtained by running xournalpp --version or in the About window
    Version of Xournal++: 1.2.3
        can be obtained by running xournalpp --version or in the About window
    Installation method: official release 

Running `xournalpp` in the terminal and then attempting to take a screenshot returns:
```
df@spooky:~$ xournalpp 
Gtk-Message: 16:53:12.418: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load it.
xopp-Message: 16:53:12.498: TEXTDOMAINDIR = /snap/xournalpp/94/usr/share/locale, Platform-specific locale dir = /usr/share/xournalpp/../locale, chosen directory = /snap/xournalpp/94/usr/share/locale
ALSA lib conf.c:4110:(config_file_load) cannot stat file/directory /usr/share/alsa/cards/aliases.conf
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.default
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.default
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.front
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround21
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround21
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround40
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround41
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround50
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround51
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.surround71
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.iec958
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.iec958
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.iec958
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.hdmi
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.hdmi
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.modem
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.modem
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.phoneline
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.phoneline
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.dmix
Cannot open DB environment: Permission denied
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
xopp-Message: 16:53:12.510: Plugin "Export" UI initialized
xopp-Message: 16:53:12.510: Plugin "MigrateFontSizes" UI initialized
xopp-Message: 16:53:12.513: Plugin "QuickScreenshot" UI initialized
Missing screenshot utility.
Please install one of:
scrot
```
I installed `scrot` and it does take a screenshot, but attempting to use the plugin within Xournal++ returns the same.

I'm a total Lua noob, but I took a look at the [plugin code](https://github.com/xournalpp/xournalpp/blob/master/plugins/QuickScreenshot/main.lua) and nothing immediately jumps out at me.

**Comment by rolandlo** [comment]
@defarm You use the snap package version. Due to the sandboxing it won't find `scrot`.
